site2/website/blog/2021-12-21-apache-pulsar-hits-10000-github-stars.md

```diff
@@ -0,0 +1,58 @@
+---
+author: Alice Bi
+title: Apache Pulsar Hits 10,000 GitHub Stars
+---
+
+Apache Pulsar hits 10,000 GitHub stars. Developers use GitHub stars to show their support for projects and to bookmark projects they want to follow. It is an important measurement to track the engagement of an open source project. The Pulsar community would like to thank every stargazer for joining us in the journey. More importantly, thank you to every Pulsar user, contributor, and committer for making this happen!
+
+Pulsar was initially developed as a cloud-native distributed messaging system at Yahoo! in 2012. Over the past decade, Pulsar has evolved into a unified messaging and streaming platform for event-driven enterprises at scale. In this blog, we look at Pulsar’s community growth, project updates, ecosystem developments, and what’s next for the project. 
+
+# Community Growth
+
+Pulsar’s success depends on its community. As shown in the chart below, the growth of Pulsar’s GitHub stars accelerated after Pulsar became a top-level Apache Software Foundation project. 
+
+![](https://i.imgur.com/kwxz2XT.png)
+
+The number of contributors overtime is another metric for measuring community engagement. The chart below shows that the number of Pulsar contributors accelerated when Pulsar became a top-level Apache project and the growth rate has continued into 2021. 
+
+![](https://i.imgur.com/TYy4CQg.png)
+
+[Pulsar surpassed Kafka in the number of monthly active contributors in early 2021](https://www.apiseven.com/en/contributor-graph?chart=contributorMonthlyActivity&repo=apache/pulsar,apache/kafka). This shows that the development and engagement of Pulsar has grown rapidly over the past few years. 
+
+![](https://i.imgur.com/6AN3dtM.png)
+
+# Project Updates and Ecosystem Development
+
+Recent Pulsar project updates have brought new capabilities to the project. Below we look at key releases and launches.  
+
+## 1. Pulsar 2.8 - Unified Messaging and Streaming with Transactions
+
+Pulsar 2.8 (released in June 2021) introduced many major updates. The Pulsar Transaction API was added to support atomicity across multiple topics and enable end-to-end exactly-once message delivery guarantee for streaming jobs. Replicated subscriptions was added to enhance Pulsar’s geo-replication. With this feature, a consumer can restart consuming from the failure point in a different cluster in case of failover. 
+
+## 2. Function Mesh - Simplifying Complex Streaming Jobs in the Cloud
```

Comment:
what is this ? it looks like it is a StreamNative product.
I agree that we can cite this, but then we should have some space to tell about other recently released projects in the Pulsar ecosystem.

The same applies to the other products cited below